doc/sphinx/Pacemaker_Explained/cluster-options.rst

```diff
@@ -62,6 +62,39 @@ Normally, you will use command-line tools that abstract the XML, so the
 distinction will be unimportant; both properties and options are cluster
 settings you can tweak.
 
+.. note::
+
+   If there are multiple ``nvpair`` options with the same name with the same
+   parent element, the first one is applied and the rest are ignored. Similarly,
+   if there are multiple options with the same name in a group of parent
+   elements with the same score within a given grandparent element, the first
+   one is applied and the rest are ignored.
+
+   This applies to parent elements ``cluster_property_set``,
+   ``instance_attributes``, ``meta_attributes``, and ``utilization`` anywhere
+   they may occur.
```

Comment:
I would lead with this, that this section describes the precedence of options in such blocks. In addition to what's here, we first need to mention how scores work with them (I don't believe that's documented yet), and give a link to the rules section.

Reply:
thought about that, was hoping I could skate by without turning this into a doc PR and chasing down all the options concepts that are undocumented

may get back to it after EOD

Reply:
I think we just need an options table for them with id and score, and a pointer to rules